.ci/component_descriptor

```diff
@@ -23,21 +23,12 @@ function buildComponentArchive() {
   CA_PATH="$(mktemp -d)"
   printf "> Building component ${COMPONENT_NAME}\n"
 
-  component-cli ca resources add ${CA_PATH} \
+  component-cli ca "${CA_PATH}" "${CTF_PATH}" \
```

Comment:
```suggestion
  component-cli ca create "${CA_PATH}" "${CTF_PATH}" \
```

Reply:
this is right. but the we need to wait for the new component-cli version to the reach concourse.